docs/UserGuide.md

```diff
@@ -45,10 +46,13 @@ refer to the image below for clarity.
 ## Getting Started
 
 You can get started with Wingman in just 3 simple steps:
-1. Click [here](https://github.com/AY2223S2-CS2103T-W11-1/tp/releases/tag/v1.3) to download the Wingman.jar file
-2. Open the terminal or command prompt (for Windows users) application in your device
-3. Run the following command in your terminal under the directory: `java -jar Wingman.jar`
+1. Ensure you have `JAVA 11` installed on your computer. If you are unsure how to check JAVA version on your computer,
+please check out [FAQ](#faq). 
+2. Click [here](https://github.com/AY2223S2-CS2103T-W11-1/tp/releases/tag/v1.3) to download the Wingman.jar file
+3. Open a command terminal on your computing, and run the command to start the application:  `java -jar Wingman.jar`
 
+You should be able to see a window popping up, and that is our Wingman application. 
+The app already contains some sample data for you to try out our features!
```

Comment:
I think you can remove this line since I've addressed this in the callout I added in my PR

Reply:
Removed. "You should be able to see a wind..."